xAPI-About.md

```diff
@@ -515,28 +509,13 @@ those listed in this specification can be implemented and co-exist with the reso
 
 <a name="COPs"></a>
 
-## 7.0 Profiles, Vocabularies, and Communities of Practice
+## 7.0 Profiles and Communities of Practice
 
-xAPI strictly defines the structure of Statements, but is very flexible as to the contents of that structure. For example, the 
-specification requires that all Statements have a "verb" property, but does not restrict the values of that property; any Verb 
-can be used. This flexibility enables xAPI to be used in any context, including future use cases not envisaged by the 
-specification authors.
+xAPI strictly defines the structure of Statements, but is very flexible as to the contents of that structure. For example, the specification requires that all Statements have a "verb" property, but does not restrict the values of that property; any Verb can be used. This flexibility enables xAPI to be used in any context, including future use cases not envisaged by the specification authors. CoPs are highly recommended to avoid duplication of effort, as creating too many ways to solve the same problem will cause fragmentation in similar domains and can potentially hurt interoperability. 
 
-It is intended that [Communities of Practice (CoPs)](#def-community-of-practice) will define Verbs, Activity types, 
-contextual relationships, extensions, etc. 
-to be used in their profiles by providing unique identifiers wherever applicable. The CoP will define these identifiers and 
-their metadata in a [Vocabulary](#def-vocabulary). A profile is a set of rules and vocabularies to implemented in addition to 
-xAPI for the particular use case being addressed. It is very important that such communities exist and share best practices. 
-For more information on publishing vocabularies, see the [Vocabulary Companion Specification](https://adl.gitbooks.io/companion-specification-for-xapi-vocabularies/content) and [Vocabulary Primer](https://adl.gitbooks.io/experience-xapi-vocabulary-primer/content/) documents.
+It is intended that [LRP](#def-learning-record-provider developers) and[CoPs](#def-community-of-practice) will create and reuse [Profiles](#def-profile) of xAPI. A profile is simply the documentation of application-specific [vocabulary](#def-vocabulary), extensions, and statement templates for the particular use case being addressed. For more information on publishing Profiles, see the [Profile Companion Specification](https://github.com/adlnet/xapi-profiles).
```

Comment:
should `[LRP](#def-learning-record-provider developers)` be `[LRP](#def-learning-record-provider) developers`?

Reply:
Yes! Good catch. Will fix this and resubmit.